### DISCUSSION TITLE
Sanic ASGI warning when running as prescribed by docs

### DISCUSSION THREAD

**Discussion by JamesHutchison** [idom-team/idom#1150] [Problem]
The docs currently recommend running via Sanic directly.

https://reactpy.dev/docs/guides/getting-started/running-reactpy.html#running-with-sanic

However, if you do, you get this warning:

`No scope. Sanic may not be running with an ASGI server`

It seems like if users should be using the built-in Sanic server then this warning should be remove, as it is emitted by ReactPy here:

https://github.com/reactive-python/reactpy/blob/main/src/py/reactpy/reactpy/backend/sanic.py#L167

If users should be using ASGI to serve Sanic, then instructions on how to do that should be provided.

**Comment by yunxiu0621** [comment]
I got this too, it seems not effect reactpy work.
